test/SlowTests/Client/TimeSeries/Replication/TimeSeriesReplicationTests.cs

```diff
@@ -927,10 +927,10 @@ public async Task PreferDeletedValues2(Options options)
                 }
 
                 await SetupReplicationAsync(storeA, storeB);
-                EnsureReplicating(storeA, storeB);
+                await EnsureReplicatingAsync(storeA, storeB);
```

Comment:
it is fixing anything? or just keep `async` all the way?

Reply:
Yes, it should fix the test, `EnsureReplicating` sends one marker, but `EnsureReplicatingAsync` considers all shards (I also wrote it in the PR message :-)).

Reply:
Shouldn't we fix `EnsureReplicating` as well then?